index.html

```diff
@@ -88,5 +88,170 @@ <h1>
+          alt="Snapshoot"
+        />
+        <h2>Multi-Post Stories</h2>
+        <div class="flex_list">
+          <ul class="list">
+            <h5>CANOPY</h5>
+
+            <li class="list_items">Back End Dev</li>
+            <li class="list_items">2015</li>
+          </ul>
+          <p class="project_description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li><a href="" class="tag">html</a></li>
+              <li><a href="" class="tag">css</a></li>
+              <li><a href="" class="tag">javascript</a></li>
+            </ul>
+          </div>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="third_project">
+        <img class="Snapshoot" src="assets/images/Tonic_2.png" alt="Snapshoot" />
+        <h2>Tonic</h2>
+        <div class="flex_list">
+          <ul class="list">
+            <h5>CANOPY</h5>
+
+            <li class="list_items">Back End Dev</li>
+            <li class="list_items">2015</li>
+          </ul>
+          <p class="project_description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li><a href="" class="tag">html</a></li>
+              <li><a href="" class="tag">css</a></li>
+              <li><a href="" class="tag">javascript</a></li>
+            </ul>
+          </div>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="forth_project">
+        <img class="Tonic" src="assets/images/Multi_Post.png" alt="Snapshoot" />
+        <h2>Multi-Post Stories</h2>
+        <div class="flex_list">
+          <ul class="list">
+            <h5>CANOPY</h5>
+
+            <li class="list_items">Back End Dev</li>
+            <li class="list_items">2015</li>
+          </ul>
+          <p class="project_description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li><a href="" class="tag">html</a></li>
+              <li><a href="" class="tag">css</a></li>
+              <li><a href="" class="tag">javascript</a></li>
+            </ul>
+          </div>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="About_Section">
+      <h2 class="heading-2">About <br />Myself</h2>
+      <p>
+        Hello I'm a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <div class="links-text">Let's connect</div>
```

Comment:
- [ ] it is the best practices to use the **Heading tag** for this element over div, :+1: :+1: 
   I think it almost certain that using semantically better markup ```<header>``` will help search engine crawlers construct a better image of your website.